__tests__/app.test.js

```diff
@@ -300,23 +300,56 @@ describe('/api/users', () => {
 });
 
 describe('/api/comments/:comment_id', () => {
-    test('DELETE request responds with 204 and the deleted comment', () => {
+    test('GET request responds with 200 and the comment', () => {
         return request(app)
-        .delete('/api/comments/2')
-        .expect(204)
+        .get('/api/comments/2')
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.comment).toMatchObject({
+                comment_id: 2,
+                body: "The beautiful thing about treasure is that it exists. Got to find out what kind of sheets these are; not cotton, not rayon, silky.",
+                votes: 14,
+                author: "butter_bridge",
+                article_id: 1,
+                created_at: expect.any(String),
+            })
+        })
+    });
+
+    test('GET request responds with 404 if comment id is valid but does not exist', () => {
+        return request(app)
+        .get('/api/comments/12345')
+        .expect(404)
+        .then(({ body }) => {
+            expect(body.msg).toBe('comment does not exist')
+        })
+    });
+
+    test('GET request responds with 400 if invalid comment_id given', () => {
+        return request(app)
+        .get('/api/comments/roar')
+        .expect(400)
         .then(({ body }) => {
-            console.log(body, '<test')
-            expect(body).toMatchObject({})
-            //     {
-            //     comment_id : 1,
-            //     body: "Oh, I've got compassion running out of my nose, pal! I'm the Sultan of Sentiment!",
-            //     votes: 16,
-            //     author: "butter_bridge",
-            //     article_id: 9,
-            //     created_at: expect.any(String),
-            //   })
+            expect(body.msg).toBe('invalid input')
         })
     });
+});
+
+
+describe('/api/comments/:comment_id', () => {
+    test('DELETE request responds with 204 and the deleted comment and GET request responds with 404 when trying to find said deleted comment', () => {
+        return request(app)
+        .delete('/api/comments/2')
+        .expect(204)
+        .then(() => {
+            return request(app)
+            .get('/api/comments/2')
+            .expect(404)
+            .then(({ body }) => {
+                expect(body.msg).toBe('comment does not exist')
+            })
+        });
```

Comment:
This should not be within here and there is already a test for a 404 below this test as well so this can simply be removed.